hail/python/hail/docs/hailpedia/matrix_table.rst

```diff
@@ -65,10 +65,10 @@ operations are allowed for a given expression.
 When evaluated in a Python interpreter, we can see ``mt.locus`` is a
 :class:`.LocusExpression` with type ``locus<GRCh37>``.
 
-    >>> mt
+    >>> mt  # doctest: +NOTEST
     <hail.matrixtable.MatrixTable at 0x1107e54a8>
 
-    >>> mt.locus
+    >>> mt.locus  # doctest: +NOTEST
```

Comment:
we should be able to leave this one on since it doesn't reference a pointer value, right?

Reply:
yeah, we could. see comment below